doc/lightweight-styling.md

```diff
@@ -34,6 +43,7 @@ All interactive controls have multiple states, such as **PointerOver** (mouse is
 	</Button.Resources>
 </Button>
 
+<!-- #3 -->
```

Comment:
@kazo0 should we also add an example to explain how to override resources using themes?

Reply:
Yes, this will likely be a common scenario.

Reply:
An example using Alias like here would be good also I think
similar to https://learn.microsoft.com/fr-fr/windows/apps/design/style/xaml-styles#aliasing-control-resources-to-new-names